system/coverage/data/CoverageGenerator.cfc

```diff
@@ -38,7 +38,7 @@ component accessors=true {
 		}
 
 		// for file globbing
-		variables.pathPatternMatcher = new PathPatternMatcher();
+		variables.pathPatternMatcher = new testbox.system.util.PathPatternMatcher();
```

Comment:
Is it ok to reach into a package sub-folder like this?  [NPM's package.json has a "main" field](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files), but I don't see anything like that in box.json.  Renaming or moving these files would be a breaking change for clients and that could be managed with semver, but it would be nice to be able to organize the package interior without causing that to be necessary.